### ISSUE TITLE
chzzk.naver.com: detection

### ISSUE THREAD

**Issue by redraoh** [uBlockOrigin/uAssets#23782]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [X] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://chzzk.naver.com/`

### Category

detection

### Description

This place is a kind of broadcast platform in Korea. When I try to chat it blocks chatroom. it unblocks me after 3-5mins disabled the extension.

Try few refresh if chatbox seems work normally. account may required to encounter those blocks.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

<img width="1790" alt="스크린샷 2024-05-21 오후 7 39 27" src="https://github.com/uBlockOrigin/uAssets/assets/19765656/7bae3774-efdc-46fa-92ce-7dcdf9d4919a">

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.57.0
Chromium: 123
filterset (summary):
 network: 137421
 cosmetic: 50187
 scriptlet: 21143
 html: 0
listset (total-discarded, last-updated):
 default:
  user-filters: 8-0, never
  KOR-1: 4966-47, now
  easylist: 87324-586, now
  easyprivacy: 50953-819, now
  plowe-0: 3726-0, now
  ublock-badware: 8734-0, now
  ublock-filters: 38327-287, now
  ublock-privacy: 924-0, now
  ublock-quick-fixes: 151-0, now
  ublock-unbreak: 2278-0, now
  urlhaus-1: 13250-0, now
filterset (user): [array of 4 redacted]
trustedset:
 added: [array of 6 redacted]
switchRuleset:
 added: [array of 5 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 208 ms (selfie)
 maxAssetCacheWait: 97 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 9
 network:
  naver.com: 9
 extended:
  ##+js(adjust-setTimeout, noPingPong||(e.sendPing, *, 0.001)
  ##+js(set-constant, getAdcrUrl, )
  ##+js(set-cookie, ba.uuid, 0, , domain, naver.com)
```
</details>

**Comment by redraoh** [comment]
try few refresh if chatbox seems work normally. account may required to encounter those blocks

**Comment by piquark6046** [comment]
Duplicate of https://github.com/List-KR/List-KR/issues/928

**Comment by stephenhawk8054** [comment]
@redraoh Please continue in the regional list repo thread above.
